aioz/chain.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "aioz",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "AIOZ Network",
+  "chain_id": "aioz_168-1",
+  "bech32_prefix": "aioz",
```

Comment:
Please register this prefix here: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Already registered. Thank you.